install/InstallTools.py

```diff
@@ -4552,6 +4552,11 @@ def install(self, baseinstall=True, mount_dirs=True, portmap=True):
 
         if not self.container_exists:
 
+            #First Make sure the latest docker image  is present
+            runImageUpdatecmd=Tools.text_replace("docker image pull {IMAGE}",args=args)
```

Comment:
Please run `black` to properly format the code. And use snake case for variables as per our guidelines.
https://github.com/threefoldtech/home/blob/master/contribution/python_projects.md

Reply:
Sorry for that, was too fast in pushing the fix